README.md

```diff
@@ -1,25 +1,27 @@
 # compliance-operator
 
-The compliance-operator is a OpenShift/Kubernetes Operator that runs an
-openscap container in a pod on nodes in the cluster.
+The compliance-operator is a OpenShift/Kubernetes Operator that allows and
```

Comment:
Considering that we use the MCO I think it would be more accurate to only say OpenShift operator